commons/src/connections/threads.ts

```diff
@@ -25,7 +25,9 @@ export const fetchThreads = async (query: MailboxQuery): Promise<Thread[]> => {
   return await fetch(queryString, getFetchConfig(query))
     .then((response) => handleResponse<MiddlewareResponse<Thread[]>>(response))
     .then((json) => {
-      return json.response;
+      return json.response.filter((thread) =>
+        thread && thread.messages ? thread?.messages?.length > 0 : false,
```

Comment:
Can this be simplified?
```suggestion
        thread?.messages?.length > 0
```

Reply:
This seems to give me a TS error